meta/flag-references/de.md

```diff
@@ -0,0 +1,237 @@
+[flag_AO]: /wiki/shared/flag/AO.gif "Angola"
+[flag_AR]: /wiki/shared/flag/AR.gif "Argentinen"
+[flag_AS]: /wiki/shared/flag/AS.gif "Amerikanisch-Samoa"
+[flag_AT]: /wiki/shared/flag/AT.gif "Österreich"
+[flag_AU]: /wiki/shared/flag/AU.gif "Australien"
+[flag_AW]: /wiki/shared/flag/AW.gif "Aruba"
+[flag_AX]: /wiki/shared/flag/AX.gif "Aland"
+[flag_AZ]: /wiki/shared/flag/AZ.gif "Azerbaidschan"
+[flag_BA]: /wiki/shared/flag/BA.gif "Bosnien und Herzegowina"
+[flag_BB]: /wiki/shared/flag/BB.gif "Barbados"
+[flag_BD]: /wiki/shared/flag/BD.gif "Bangladesch"
+[flag_BE]: /wiki/shared/flag/BE.gif "Belgien"
+[flag_BF]: /wiki/shared/flag/BF.gif "Burkina Faso"
+[flag_BG]: /wiki/shared/flag/BG.gif "Bulgarien"
+[flag_BH]: /wiki/shared/flag/BH.gif "Bahrain"
+[flag_BI]: /wiki/shared/flag/BI.gif "Burundi"
+[flag_BJ]: /wiki/shared/flag/BJ.gif "Benin"
+[flag_BL]: /wiki/shared/flag/BL.gif "Saint-Barthelemy"
+[flag_BM]: /wiki/shared/flag/BM.gif "Bermuda"
+[flag_BN]: /wiki/shared/flag/BN.gif "Brunei"
+[flag_BO]: /wiki/shared/flag/BO.gif "Bolivien"
+[flag_BR]: /wiki/shared/flag/BR.gif "Brasilien"
+[flag_BS]: /wiki/shared/flag/BS.gif "Bahamas"
+[flag_BT]: /wiki/shared/flag/BT.gif "Bhutan"
+[flag_BW]: /wiki/shared/flag/BW.gif "Botswana"
+[flag_BY]: /wiki/shared/flag/BY.gif "Belarus"
+[flag_BZ]: /wiki/shared/flag/BZ.gif "Belize"
+[flag_CA]: /wiki/shared/flag/CA.gif "Kanada"
+[flag_CD]: /wiki/shared/flag/CD.gif "Demokratische Republik Kongo"
+[flag_CF]: /wiki/shared/flag/CF.gif "Zentralafrikanische Republik"
+[flag_CG]: /wiki/shared/flag/CG.gif "Kongo"
+[flag_CH]: /wiki/shared/flag/CH.gif "Schweiz"
+[flag_CI]: /wiki/shared/flag/CI.gif "Elfenbeinküste"
+[flag_CL]: /wiki/shared/flag/CL.gif "Chile"
+[flag_CM]: /wiki/shared/flag/CM.gif "Kamerun"
+[flag_CN]: /wiki/shared/flag/CN.gif "China"
+[flag_CO]: /wiki/shared/flag/CO.gif "Kolumbien"
+[flag_CR]: /wiki/shared/flag/CR.gif "Costa Rica"
+[flag_CU]: /wiki/shared/flag/CU.gif "Kuba"
+[flag_CV]: /wiki/shared/flag/CV.gif "Kap Verde"
+[flag_CX]: /wiki/shared/flag/CX.gif "Weihnachtsinsel"
+[flag_CY]: /wiki/shared/flag/CY.gif "Republik Zypern"
+[flag_CZ]: /wiki/shared/flag/CZ.gif "Tschechien"
+[flag_DE]: /wiki/shared/flag/DE.gif "Deutschland"
+[flag_DJ]: /wiki/shared/flag/DJ.gif "Dschibuti"
+[flag_DK]: /wiki/shared/flag/DK.gif "Dänemark"
+[flag_DM]: /wiki/shared/flag/DM.gif "Dominica"
+[flag_DO]: /wiki/shared/flag/DO.gif "Dominikanische Republik"
+[flag_DZ]: /wiki/shared/flag/DZ.gif "Algerien"
+[flag_EC]: /wiki/shared/flag/EC.gif "Ecuador"
+[flag_EE]: /wiki/shared/flag/EE.gif "Estland"
+[flag_EG]: /wiki/shared/flag/EG.gif "Ägypten"
+[flag_ER]: /wiki/shared/flag/ER.gif "Eritrea"
+[flag_ES]: /wiki/shared/flag/ES.gif "Spanien"
+[flag_ET]: /wiki/shared/flag/ET.gif "Äthiopien"
+[flag_EU]: /wiki/shared/flag/EU.gif "Europäische Union"
+[flag_FI]: /wiki/shared/flag/FI.gif "Finnland"
+[flag_FJ]: /wiki/shared/flag/FJ.gif "Fidschi"
+[flag_FK]: /wiki/shared/flag/FK.gif "Falklandinseln (Malvinas)"
```

Comment:
```suggestion
[flag_FK]: /wiki/shared/flag/FK.gif "Falklandinseln (Malvinen)"
```

Reply:
[This source](https://www.garantiert-reisen.de/reiseinformationen-falklandinseln/) writes it `Malwinen`.

Reply:
oh yeah [wikipedia](https://de.wikipedia.org/wiki/Falklandinseln) does aswell. my mistake, i updated the suggestion